--- a/Makefile	2020-06-04 19:43:20.582191740 -0600
+++ b/Makefile	2020-06-04 19:42:27.698185701 -0600
@@ -41,7 +41,7 @@
 
 # builds kind in a container, outputs to $(OUT_DIR)
 kind:
-	hack/go_container.sh go build -v -o /out/$(KIND_BINARY_NAME) -ldflags "$(LD_FLAGS)"
+	go build -v -o $(OUT_DIR)/$(KIND_BINARY_NAME) -ldflags "$(LD_FLAGS)"
 
 # alias for building kind
 build: kind
@@ -53,7 +53,7 @@
 
 # cleans the cache volume
 clean-cache:
-	$(DOCKER) volume rm -f kind-build-cache >/dev/null
+	rm -f kind-build-cache >/dev/null
 
 # cleans the output directory
 clean-output:
