diff --git a/CMakeLists.txt b/CMakeLists.txt
index e851dda..fb14535 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -153,7 +153,7 @@ endif()
 if(UNIX AND NOT APPLE)
 	include(GNUInstallDirs)
 
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -mtune=core2 -Ofast")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Ofast")
 
 	set_target_properties(obs-ndi PROPERTIES PREFIX "")
 	target_link_libraries(obs-ndi obs-frontend-api)
